Tests/PointFreeTests/PricingTests.swift

```diff
@@ -35,9 +35,9 @@ class PricingTests: TestCase {
         
         webView.evaluateJavaScript(
           """
-          document.getElementById("tab0").checked = false;
-          document.getElementById("tab1").checked = true;
-          var quantity = document.getElementsByName("pricing[team]")[0];
+          document.getElementById('tab0').checked = false;
+          document.getElementById('tab1').checked = true;
+          var quantity = document.getElementsByName('pricing[quantity]')[0];
```

Comment:
lol I heard that single quotes are more performant than double quotes......

jkjk

Reply:
Lolol we use single quotes elsewhere so...consistency dragging me. Makes it at least nice to do non-multiline JS where we'd have to escape all `"`s.